code/processes/wdb.q

```diff
@@ -430,7 +430,14 @@ subscribe:{[]
 		/- return the tables subscribed to and the tickerplant log date
 		subto:.sub.subscribe[subtabs;subsyms;schema;replay;subproc];
 		/- check the tp logdate against the current date and correct if necessary 
-		fixpartition[subto];];}
+		fixpartition[subto];];
+		/-retrieving table stripe mapping
+		stphandle:first exec w from s;
+		.ds.tblstripe:@[stphandle;"select tbl,filts from .stpps.subrequestfiltered where handle = .z.w";
```

Comment:
oh i missed that, will have a look into it

Reply:
From what I can tell when it comes to the tailer/rdb, `.stpps.subrequestfiltered` gets defined by the pubsub.q script rather than being passed with `subto`.  It appears empty in the tailer/rdb after being defined too, since the table tracks the mapping of the subscribers' tables, so it doesn't contain the info that would be needed.

Reply:
Sure - my point is does `subto` contain the information we need here? From what I remember, we made it so that when a process does a striped subscription, STP will return the filter that is being used as part of the returned object (i.e. `subto` here), so that the subscriber knows what filter to apply when replaying TP logs etc. So it seems like we should already have that information without having to re-request it manually etc.

It's possible we changed that & `subto` doesn't contain the filter, in which can ignore this.

Reply:
It doesn't appear to contain a filter anymore.  `subtabs` and `subsyms` contained in `subto` are set to all, so the datastriping filter must be applied elsewhere. The other three args are set to a true boolean - `schema`,`replay` and `subproc`. Their functionality doesn't indicate a filter either but rather just to retrieve schema from the TP and replay the TP log file.

Reply:
OK fair enough, guess this is fine

Reply:
I can't see any striping info in the returned object too